packages/toc/src/tokens.ts

```diff
@@ -225,6 +225,14 @@ export namespace TableOfContents {
      */
     isActive: boolean;
 
+    /**
+     * Set a new active heading.
+     *
+     * @param heading The new active heading
+     * @param emitSignal Whether to emit the activeHeadingChanged signal or not.
+     */
+    setActiveHeading(heading: H | null, emitSignal?: boolean): void;
```

Comment:
Note to self: since this is a follow-up to https://github.com/jupyterlab/jupyterlab/pull/12374 we don't need to document the change in the public interface.